src/SideciTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Yoshihiro
+ * Date: 4/22/16
+ * Time: 11:24 PM
+ */
+
+namespace OhSky\LineTrialBot;
+
+
+class SideciTest{
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition